index.html

```diff
@@ -0,0 +1,46 @@
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+        <nav id="nav-bar"></nav>
+        <img id="header-img" src="Images/fruits..jpeg" alt=" This is a picture of fruits" />
+        <nav id="nav-bar" class="sticky">
+            <ul>
+                <li><a class="nav-link" href="#features">Features</a></li>
+                <li><a class="nav-link" href="#how_it_works">How It Works</a></li>
+                <li><a class="nav-link" href="#pricing">Pricing</a></li>
+            </ul>
+        </nav>
+        <iframe id="video" width="560" height="315" src="https://www.youtube.com/embed/S0S1M9bKlYk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+    </header>
+
+    <div class="container"></div>
```

Comment:
This line was wrong.
/*<div class-"container">


</div>*/

Reply:
Hi, welldone! Please kindly ensure that your comment is descriptive enough in a clear and polite manner and always state the reason(s) for the required changes. It might be obvious to you and not so to others. 👍